packages/datetime/src/common/dateUtils.ts

```diff
@@ -107,3 +109,21 @@ export function getDateTime(date: Date, time: Date) {
             time.getHours(), time.getMinutes(), time.getSeconds(), time.getMilliseconds());
     }
 }
+
+export function getDatePreviousMonth(date: Date): Date {
+    const monthIsJanuary = date.getMonth() === Months.JANUARY;
+    if (monthIsJanuary) {
```

Comment:
nit: IMO the equality expression above is clear enough without this extra variable.